README.md

```diff
@@ -46,13 +46,13 @@ Copy the contents and remove the whitelines, `-----BEGIN PRIVATE KEY-----` and `
 Use this private key together with the issuer ID and the private key ID to create your configuration file:
 
 ```swift
-let configuration = APIConfiguration(issuerID: "<YOUR ISSUER ID>", privateKeyID: "<YOUR PRIVATE KEY ID>", privateKey: "<YOUR PRIVATE KEY>")
+let configuration = APIConfiguration(issuerID: "<YOUR ISSUER ID>", privateKeyID: "<YOUR PRIVATE KEY ID>", privateKey: "<YOUR PRIVATE KEY>", expirationDuration: "<OPTIONAL EXPIRE DURATION>")
```

Comment:
Since it's optional and likely not used by everyone, I think it's better to keep the minimum configuration sample in the readme. Thoughts?

```suggestion
let configuration = APIConfiguration(issuerID: "<YOUR ISSUER ID>", privateKeyID: "<YOUR PRIVATE KEY ID>", privateKey: "<YOUR PRIVATE KEY>")
```